src/ImpactX.cpp

```diff
@@ -11,10 +11,11 @@
 #include <AMReX.H>
 #include <AMReX_REAL.H>
 #include <AMReX_ParmParse.H>
+#include <AMReX_Utility.H>
 
 #include <string>
 #include <vector>
-
+#include <math.h>
```

Comment:
That's the preferred name in C++ for the same functionality:
```suggestion
#include <cmath>
```